app/models/menu_order.rb

```diff
@@ -0,0 +1,60 @@
+class MenuOrder
```

Comment:
A bit of a nit pick but since it is more specific to accounts, I think it should be called `AccountOrder`